dockerfile_parse/parser.py

```diff
@@ -66,11 +66,13 @@ def __setitem__(self, key, value):
 
 
 class DockerfileParser(object):
-    def __init__(self, path=None, cache_content=False, env_replace=True):
+    def __init__(self, path=None, cache_content=False, env_replace=True, parent_env={}):
```

Comment:
Please use `parent_env=None`. Using non-primitive types as default values is quite dangerous.